lib/Doctrine/ORM/Query/AST/Functions/ConcatFunction.php

```diff
@@ -64,17 +60,11 @@ public function parse(\Doctrine\ORM\Query\Parser $parser)
         $parser->match(Lexer::T_IDENTIFIER);
         $parser->match(Lexer::T_OPEN_PARENTHESIS);
         
-        $this->firstStringPrimary = $parser->StringPrimary();
-        $this->concatExpressions[] = $this->firstStringPrimary;
-        
-        $parser->match(Lexer::T_COMMA);
-        
-        $this->secondStringPrimary = $parser->StringPrimary();
-        $this->concatExpressions[] = $this->secondStringPrimary;
+        $this->concatExpressions[] = $parser->StringPrimary();
```

Comment:
All of the changes in this file are unnecessary - DQL CONCAT already supported >2 arguments - The semantics *required* at least 2 arguments. This change makes it so you can do weird things like CONCAT(field) - That isn't really valid. CONCAT should require a minimum of 2 arguments.